web/src/main/resources/static/oncotree-mapping.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html;charset=utf-8"/>
+    <link type="text/css" rel="stylesheet" href="css/bootstrap.css"/>
+    <link type="text/css" rel="stylesheet" href="css/style.css"/>
+    <script type="text/javascript" src="js/jquery-1.11.1.min.js"></script>
+    <script type="text/javascript" src="js/bootstrap.js"></script>
+
+  </head>
+
+  <body>
+ 
+    
+
+<div class="container">
+
+	<div class="row">
+	
+		<div class="col-md-8">
+			<h3><u>Oncotree Mappings</u></h3>
+			
+			<p>
+			You have the following two ways of getting mappings to Oncotree
+			</p>
+			
+			<dl>
+			  
+			  <dt>Via Vocabulary and a Concept IDs</dt>
+			  <dd>Choose a vocabulary ID from the dropdown, e.g. ICD10, and input a concept ID, e.g. C92 (Acute Myeloid Leukemia )</dd>
+			  </br>
+			  <dt>Via ICDO Site and Histology Codes</dt>
+			  <dd>Pick ICDO from the vocabulary dropdown, and enter a Histology Code, e.g. 8490/3, and a Site Code, e.g. C16.1</dd>
+			  
+			</dl>
+			
+			<address>
+			  <strong>If You have any questions contact </strong>
+			  <a href="mailto:#">huergasi@mskcc.org</a>

```

Comment:
This may be a spam invitation on a public website ... I might recommend we disguise/mask/imagify the email address.